x-pack/plugins/security_solution/public/overview/components/detection_response/recent_cases_table/recent_cases_table.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useMemo } from 'react';
+
+import {
+  EuiBasicTable,
+  EuiBasicTableColumn,
+  EuiButton,
+  EuiEmptyPrompt,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import { CaseStatuses } from '@kbn/cases-plugin/common';
+
+import { SecurityPageName } from '../../../../app/types';
+import { FormattedDate } from '../../../../common/components/formatted_date';
+import { HeaderSection } from '../../../../common/components/header_section';
+import { HoverVisibilityContainer } from '../../../../common/components/hover_visibility_container';
+import { BUTTON_CLASS as INPECT_BUTTON_CLASS } from '../../../../common/components/inspect';
+import { CaseDetailsLink } from '../../../../common/components/links';
+import { useQueryToggle } from '../../../../common/containers/query_toggle';
+import { useNavigation, NavigateTo, GetAppUrl } from '../../../../common/lib/kibana';
+import * as i18n from '../translations';
+import { LastUpdatedAt } from '../util';
+import { StatusBadge } from './status_badge';
+import { RecentCaseItem, useRecentlyCreatedCases } from './use_recent_cases_items';
+
+type GetTableColumns = (params: {
+  getAppUrl: GetAppUrl;
+  navigateTo: NavigateTo;
+}) => Array<EuiBasicTableColumn<RecentCaseItem>>;
+
+const DETECTION_RESPONSE_RECENT_CASES_QUERY_ID = 'recentlyCreatedCasesQuery';
+
+export const RecentlyCreatedCasesTable = React.memo(() => {
```

Comment:
there's a discrepancy between the file and folder name and the component name. They should have the same name. But also, I would be a little bit more agnostic with the name, what if at some point product decides to sort this table by the number of alerts, severities, or any other criteria instead of timestamp, they would not be "recently created" cases anymore and we'd have to change its name. 
Since we are in the _detection_response_ folder, you can give it a simple name like `CasesTable` (there's only one cases table in the dashboard).

Reply:
changed!